src/google_apps_clj/google_sheets_v4.clj

```diff
@@ -240,8 +241,9 @@
                                     (.setGridProperties
                                      (-> (GridProperties.)
                                          (.setRowCount (int (count rows)))
-                                         (.setColumnCount (int num-cols)))))))))
-                      (-> (Request.)
+                                          (.setColumnCount (int num-cols)))))))))]
+                      (when (< 0 (count rows))
```

Comment:
Not afaict, it's just purportedly idiomatic clojure; certainly habitual on my part by now.

FWIW `empty?` and `not-empty` are in fact implemented using `seq`.